examples/relying_party/dumps/example.json

```diff
@@ -763,6 +772,9 @@
       }
     ],
     "metadata": {
+      "federation_entity": {
+        "federation_resolve_endpoint": "http://127.0.0.1:8000/oidc/op/resolve"
```

Comment:
can I ask you to put also in federation_entity the following members?

organization_name
homepage_uri
policy_uri
logo_uri
contacts

Reply:
ok!